Setup/UpgradeSchema.php

```diff
@@ -41,20 +41,27 @@ public function upgrade(SchemaSetupInterface $setup,
             }
         }
 
-        if (version_compare($context->getVersion(), 'beta-3.3.4', '>')) {
-            $tableName = $setup->getTable('sales_order');
-            if ($setup->getConnection()->isTableExists($tableName) == true) {
-                $setup->getConnection()->addColumn(
-                    $tableName,
-                    'rzp_order_id',
-                    [
-                        'nullable' => true,
-                        'type'     => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
-                        'length'   => 55,
-                        'comment'  => 'RZP Order ID'
-                    ]
-                );
-            }
+        $tableName = $setup->getTable('sales_order');
+        if ($setup->getConnection()->isTableExists($tableName) == true) {
```

Comment:
Keep the opening braces for this if statement in new line

Reply:
@nikhilsharma1100 Updated opening braces for if statement in new line